tests/parsers/fixtures/xspectra/failed_timeout/aiida.out

```diff
@@ -0,0 +1,469 @@
+
+     Program XSpectra v.6.5 starts on  1Sep2022 at 10: 2:20 
+
+     This program is part of the open-source Quantum ESPRESSO suite
+     for quantum simulation of materials; please cite
+         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
+         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
+          URL http://www.quantum-espresso.org", 
+     in publications or presentations arising from this work. More details at
+     http://www.quantum-espresso.org/quote
+
+     Parallel version (MPI), running on    16 processors
+
+     MPI processes distributed on     1 nodes
+     R & G space division:  proc/nbgrp/npool/nimage =      16
+
+     -------------------------------------------------------------------------
```

Comment:
For this test, try to cut out as much of this file as possible while the test still works. You are just testing here that if `Calculation not finished` is found at the bottom, the parser returns the right exit code.